maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java

```diff
@@ -70,7 +70,7 @@
     implements ModelValidator
 {
 
-    private static final Pattern CI_FRIENDLY_EXPRESSION = Pattern.compile( "\\$\\{(.+?)\\}" );
+    private static final Pattern EXPRESSION_NAME = Pattern.compile( "\\$\\{(.+?)\\}" );
```

Comment:
I think the new var name does not make sense since it does not contain the name of th expression, but the espression itself.

Reply:
The first group is the expression name, which is always the thing you're interested in.

Reply:
You are right, `EXPRESSION_NAME_PATTERN`? I wonder why it was called `CI_FRIENDLY_EXPRESSION`.

Reply:
It was only used for CI-friendly expressions. I noticed I needed exactly the same regexp, hence had to rename it.

Reply:
`EXPRESSION_NAME_PATTERN` is fine by me.

Reply:
Accepted and makes sense